libs/telemetry/events/bundle_init.go

```diff
@@ -0,0 +1,23 @@
+package events
+
+// Corresponds to the `DatabricksCliBundleInitEvent` proto message in `databricks_cli_log.proto`
+// as of 20 Dec 2024.
+type BundleInitEvent struct {
```

Comment:
It seems to be bundle specific but it lives in `libs` folder, shall it be moved to `bundle/telemetry/events` package instead?

Reply:
I think it's nice to have all the type definitions colocated in the library itself since they have a corresponding proto in universe as well and are not just stand alone.

I'd be fine with either approach though.